zScanner/DomainModel/DocumentTypeDomainModel.swift

```diff
@@ -35,4 +22,4 @@ extension DocumentTypeDomainModel: ListItem {
 }
 
 // MARK: -
-typealias DocumentDict = [DocumentMode: [DocumentTypeDomainModel]]
+//typealias DocumentDict = [DocumentMode: [DocumentTypeDomainModel]]
```

Comment:
Don't leave commented code behind. Remove it if it's unused

Reply:
I agree with you. I just wasn't sure how document mode infects the app flow. So I try to remove it carefully.